code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -74,7 +74,12 @@
 	icon_state = "tomealtar"
 	luminosity = 2
 	l_color = "#3e0000"
-//	luminosity = 5
+
+/obj/structure/cult/tome/attackby(obj/item/weapon/W as obj, mob/user as mob)
+	if(isrobot(user))
```

Comment:
oh, well, ok then, I might remove it from tables_racks.dm as well since I took the code from there.